website/src/recurring-text/aria-attributes-no-list.txt

```diff
@@ -0,0 +1,3 @@
+In the component itself we use multiple aria-attributes to assure the component works well with a range of assistive technologies. For some attributes however it is not possible for the Design System to add a meaningfull value, because it relies on the context or way a component is used.
```

Comment:
Maybe it would be nice to have a `utility` directory in the `src` and then we can have inside `recurring-text` folder?